tensorflow/python/kernel_tests/pooling_ops_test.py

```diff
@@ -45,11 +45,14 @@ def GetTestConfigs(include_nchw_vect_c=False):
     all the valid test configs as tuples of data_format and use_gpu.
   """
   test_configs = [("NHWC", False), ("NHWC", True)]
-  if not test.is_gpu_available(cuda_only=True):
+  if not test.is_gpu_available():
```

Comment:
We actaully need cuda_only=True here. We do not have NCHW support as it's exclusive for CUDA